templates/s3-bucket.yaml

```diff
@@ -0,0 +1,94 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Transform: S3Objects
+
+Description: >-
+  An S3 bucket for use in development. This bucket will serve as the landing
+  bucket for incoming data, mimicking the Bridge bucket.
+
+Parameters:
+
+  BucketName:
+    Type: String
+    Description: Name of the bucket.
+    Default: ''
+
+  ConnectToSynapse:
+    Type: String
```

Comment:
Is there a boolean type here?

Reply:
Cloudformation doesn't have boolean types. This is a string representation of a boolean, which we reference in the respective Condition.